src/react/components/pages/projectSettings/projectForm.json

```diff
@@ -41,6 +41,78 @@
                 "frameExtractionRate"
             ]
         },
+        "activeLearningSettings": {
```

Comment:
I would like to see this broken down into its own form similar to export settings.  Longer term we are going to update the UI to a tabbed interface where active learning deserves its own tab - but for now let's move out into separate form.

Reply:
Completely agree with you but just wondering if we are sure we want to do this in this PR ?  I mean this is not probably the only thing should be stripped out, think about tags for example.  Why we want to start with this and why in this PR ?